### DISCUSSION TITLE
libmpv: how to get a transparent window when using gpu-context=d3d11 ?

### DISCUSSION THREAD

**Discussion by wbtcpip2** [mpv-player/mpv#14759] [Q&A]
i use:
 background=color
 background-color=#000000

when i load a transparent png with gpu-api=opengl the background is transparent like this:
![2024-08-29_020019](https://github.com/user-attachments/assets/8e23768b-19aa-45cc-a661-3f191fc53d13)

but the same with gpu-context=d3d11 is not transparent:
![2024-08-29_020044](https://github.com/user-attachments/assets/6e1697f5-d28b-4e0e-9f2d-d19c400ae45b)

is there any parameter to make d3d11 alpha transparent like opengl?

**Comment by ghost** [comment]
You have to disable flip-mode presentation with `--d3d11-flip=no`

**Comment by wbtcpip2** [comment]
oo thank you very much! it works!!

![2024-08-29_023533](https://github.com/user-attachments/assets/d27d11ce-b8ef-4d52-bda9-38fc9f60a767)

**Comment by Andarwinux** [comment]
disable flip presentation is bad for performance. Maybe eventually need to consider DirectComposition or Windows.UI.Composition.

**Comment by wbtcpip2** [comment]
thank you for advice. i will only disable d3d11-flip sporadically when i have to display transparent png otherwise i'll keep it enabled
